### PR TITLE
Use single project instead of two

### PR DESCRIPTION
**Pull request by edwardmedia**

### CODE REVIEW

File: gke-to-gke-peering/README.md [hunk 1]
```diff
@@ -62,17 +62,15 @@ The execution of this code in the GCP environment creates two custom GCP network
 
 Below is the detailed overview of GCP resources which will be created.
 
-### GCP Project 1
+### GCP Network network1
```
Comment (robinpercy) [inline gke-to-gke-peering/README.md]: This reads strangely. Should it just be "GCP Network 1"?

File: gke-to-gke-peering/README.md [hunk 2]
```diff
@@ -126,33 +121,12 @@ those clusters.
 1. Kubernetes Engine >= 1.10.0-gke.0
 
 ### Setup
-1. Pick unique names for PROJECT1 and PROJECT2 and set the variables.
```
Comment (robinpercy) [inline gke-to-gke-peering/README.md:129]: We should tell the user to set their default project if it isn't already the one they want to create the demo in.

File: gke-to-gke-peering/README.md [hunk 3]
```diff
@@ -168,11 +142,11 @@ https://cloud.google.com/compute/quotas.
 The following steps will allow a user to
 
 1. Change directory to `gke-to-gke-peering`
-1. Run `./install.sh $PROJECT1 $PROJECT2`
+1. Run `./install.sh`
 
 ## Validation
 1. Make sure that there are no errors in the install script execution.
```
Comment (robinpercy) [inline gke-to-gke-peering/README.md:149]: I think we should move these manual validation steps into a general troubleshooting section. The automated validation script will test all of it anyway. 
@chrislovecnm what do you think?

File: gke-to-gke-peering/cleanup.sh [hunk 4]
```diff
@@ -77,23 +70,23 @@ kubectl delete -f "$ROOT"/manifests/run-my-nginx.yaml
 sleep 120
 
 ### Delete clusters
-gcloud deployment-manager deployments delete cluster1-deployment --project "$1" \
+gcloud deployment-manager deployments delete cluster1-deployment \
```
Comment (robinpercy) [inline gke-to-gke-peering/cleanup.sh:73]: While we're in here, we can group all these deployments into a single command: `gcloud deployment-manager deployments delete cluster1-deployment cluster2-deployment cluster3-deployment cluster4-deployment` to speed up the script.

File: gke-to-gke-peering/cleanup.sh [hunk 5]
```diff
@@ -77,23 +70,23 @@ kubectl delete -f "$ROOT"/manifests/run-my-nginx.yaml
...
 gcloud compute networks peerings delete peer-network1-to-network2  \
-  --network network1 --project "$1" --quiet
+  --network network1 --quiet
 gcloud compute networks peerings delete peer-network2-to-network1  \
-  --network network2 --project "$2" --quiet
+  --network network2 --quiet
 
 ### Delete network
-gcloud deployment-manager deployments delete network1-deployment --project "$1" \
+gcloud deployment-manager deployments delete network1-deployment \
```
Comment (robinpercy) [inline gke-to-gke-peering/cleanup.sh:89]: Same as above, lets group as many deployment-manager calls together as possible.

File: gke-to-gke-peering/install.sh [hunk 6]
```diff
@@ -30,55 +30,41 @@ command -v kubectl >/dev/null 2>&1 || \
...
-
+### enable required service apis in the project
 gcloud services enable \
     compute.googleapis.com \
-    deploymentmanager.googleapis.com \
-    --project="$2"
+    deploymentmanager.googleapis.com 
 
 ### create networks and subnets
 gcloud deployment-manager deployments create network1-deployment \
```
Comment (robinpercy) [inline gke-to-gke-peering/install.sh:44]: Lets group all of these deployment-manager calls

File: gke-to-gke-peering/validate-pod-to-service-communication.sh [hunk 7]
```diff
@@ -18,22 +18,19 @@
 ### of the pods in cluster1.
 ### cluster zones can be modified as needed.
 
-### Validate gcp projects
-( gcloud projects describe "$1" | grep projectNumber >/dev/null 2>&1 ) || \
-	{  echo "Project 1 is not valid. Aborting."; exit 1; }
+### Obtain current active PROJECT_ID
+PROJECT_ID=$(gcloud config get-value project)
```
Comment (robinpercy) [inline gke-to-gke-peering/validate-pod-to-service-communication.sh:22]: We should validate that PROJECT_ID is not empty.

File: gke-to-gke-vpn/README.md [hunk 8]
```diff
@@ -148,17 +144,11 @@ those clusters.
 1. Kubernetes Engine >= 1.10.0-gke.0
 
 ### Setup
-1. Pick unique names for PROJECT1 and PROJECT2 and set the variables.
```
Comment (robinpercy) [inline gke-to-gke-vpn/README.md:151]: Add a step for setting or verifying the default project is the one they want to use

File: gke-to-gke-vpn/README.md [hunk 9]
```diff
@@ -173,11 +163,11 @@ those clusters.
 The following steps will allow a user to run this demo.
 
 1. Change directory to `gke-to-gke-vpn`
-1. Run `./install.sh $PROJECT1 $PROJECT2`
+1. Run `./install.sh`
 
 ## Validation
 1. Make sure that there are no errors in the install script execution.
```
Comment (robinpercy) [inline gke-to-gke-vpn/README.md:170]: As with peering, I think this can be removed.

File: gke-to-gke-vpn/cleanup.sh [hunk 10]
```diff
@@ -76,38 +71,24 @@ kubectl delete -f "$ROOT"/manifests/run-my-nginx.yaml
...
 ### Delete clusters
-gcloud deployment-manager deployments delete cluster1-deployment --project "$1" \
-  --quiet
-gcloud deployment-manager deployments delete cluster2-deployment --project "$1" \
-  --quiet
-gcloud deployment-manager deployments delete cluster3-deployment --project "$2" \
-  --quiet
-gcloud deployment-manager deployments delete cluster4-deployment --project "$2" \
-  --quiet
+gcloud deployment-manager deployments delete cluster1-deployment --quiet
```
Comment (robinpercy) [inline gke-to-gke-vpn/cleanup.sh:74]: We can group all of the these deployment-manager calls into a single call

File: gke-to-gke-vpn/install.sh [hunk 11]
```diff
@@ -27,103 +27,94 @@ command -v gcloud >/dev/null 2>&1 || \
...
-    deploymentmanager.googleapis.com \
-    --project="$1"
+	deploymentmanager.googleapis.com
 
-gcloud services enable \
-    compute.googleapis.com \
-    deploymentmanager.googleapis.com \
-    --project="$2"
+### Obtain current active PROJECT_ID
+PROJECT_ID=$(gcloud config get-value project)
```
Comment (robinpercy) [inline gke-to-gke-vpn/install.sh]: Lets test that PROJECT_ID is not empty (that the default was set and found)

File: gke-to-gke-vpn/validate-pod-to-service-communication.sh [hunk 12]
```diff
@@ -17,24 +17,20 @@
 ### Executes commands to verify pod-to-service communication from test container of the pods in cluster1
 ### cluster zones can be modified as needed.
 
-### Validate gcp projects
-( gcloud projects describe "$1" | grep projectNumber >/dev/null 2>&1 ) || \
-	{  echo "Project 1 is not valid. Aborting."; exit 1; }
+### Obtain current active PROJECT_ID
+PROJECT_ID=$(gcloud config get-value project)
```
Comment (robinpercy) [inline gke-to-gke-vpn/validate-pod-to-service-communication.sh:21]: Test that PROJECT_ID is not empty

**Review by robinpercy — COMMENTED** [review]
Looking good. Just a few minor suggestions (repeated 4x 😂 )

File: gke-to-gke-peering/README.md (hunk 3 above)
Comment (chrislovecnm) [reply gke-to-gke-peering/README.md:149]: We have a validation section in all of our readmes don’t we? I think it is in our template

File: gke-to-gke-peering/cleanup.sh (hunk 4 above)
Comment (chrislovecnm) [reply gke-to-gke-peering/cleanup.sh:73]: That is for another PR ;) That is what he is working on now. I would rather have that in a separate PR and commit.

File: gke-to-gke-peering/README.md (hunk 2 above)
Comment (edwardmedia) [reply gke-to-gke-peering/README.md:129]: Agreed. I thought about that. I will add that in the README.md

File: gke-to-gke-peering/README.md (hunk 3 above)
Comment (edwardmedia) [reply gke-to-gke-peering/README.md:149]: I do have a validation section in the README.md

File: gke-to-gke-peering/cleanup.sh (hunk 4 above)
Comment (edwardmedia) [reply gke-to-gke-peering/cleanup.sh:73]: We can have one call to "delete" all resources like mentioned. But it seems a little bit tricky to implement one call to "create" all resources. Is it that the new PR targets?

**Comment by edwardmedia** [comment]
Updated based on comments

**Review by techgnosis — APPROVED** [review]
Looks like the changes were made to the Network name and the default project instructions. /lgtm

**Pull request merged**